### PR TITLE
Clean up docs

### PR DESCRIPTION
**Pull request by thockin**
Add an example doc for cross-building.

Clean up other bits of README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,24 +4,42 @@ git-sync is a simple command that pulls a git repository into a local directory.
...
 it re-pulls, it updates the destination directory atomically.  In order to do
 this, it uses a git worktree in a subdirectory of the `--root` and flips a
 symlink.
 
-git-sync can also be configured to make webhook call upon sucessful git repo syncronisation. The call is made when right after the symlink is updated.
+git-sync can pull over HTTP(S) (with authentication or not) or SSH.
 
-## Usage
+git-sync can also be configured to make a webhook call upon sucessful git repo
+syncronization. The call is made after the symlink is updated.
```
Comment (stp-ip) [inline README.md]:
```suggestion
synchronization. The call is made after the symlink is updated.
```

File: README.md [hunk 2]
```diff
@@ -4,24 +4,42 @@ git-sync is a simple command that pulls a git repository into a local directory.
...
 re-pull if the target of the run has changed in the upstream repository.  When
 it re-pulls, it updates the destination directory atomically.  In order to do
 this, it uses a git worktree in a subdirectory of the `--root` and flips a
 symlink.
 
-git-sync can also be configured to make webhook call upon sucessful git repo syncronisation. The call is made when right after the symlink is updated.
+git-sync can pull over HTTP(S) (with authentication or not) or SSH.
 
-## Usage
+git-sync can also be configured to make a webhook call upon sucessful git repo
```
[the remark this answers is not in the document]
Comment (thockin) [reply README.md]: fixed in next push

File: README.md (hunk 1 above)
Comment (thockin) [reply README.md]: fixed in next

**Comment by thockin** [comment]
pushed fixes

**Pull request merged**